examples/python/tensor_transforms.ipynb

```diff
@@ -0,0 +1,611 @@
+        "id": "XgndWvG3cBWs"
+      },
+      "source": [
+        "# TO REMOVE ONCE v0.8.0 RELEASED\n",
+        "\n",
+        "# pip install --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html\n",
+        "\n",
+        "!pip uninstall -y -q torchvision\n",
+        "!pip install --pre -q --upgrade torch -f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html\n",
+        "!git clone https://github.com/pytorch/vision.git\n",
+        "!cd vision && python setup.py develop > /dev/null\n",
+        "\n",
+        "# restart runtime: ctrl/cmd+m+,"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "btaDWPDbgIyW",
+        "outputId": "8a83d408-f643-42da-d247-faf3a1bd3ae0",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 35
+        }
+      },
+      "source": [
+        "import torch, torchvision\n",
+        "torch.__version__, torchvision.__version__"
+      ],
+      "execution_count": 1,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "('1.7.0.dev20201001+cu101', '0.8.0a0+6b41eb0')"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 1
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "tbT1rJSrf6It"
+      },
+      "source": [
+        ""
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9Vj9draNb4oA"
+      },
+      "source": [
+        "## Transforms on CPU/CUDA tensor images\n",
+        "\n",
+        "Let's show how to apply transformations on image opened as a tensor "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Epp3hCy0b4oD"
+      },
+      "source": [
+        "!wget https://farm1.static.flickr.com/152/434505223_8d1890e1e2.jpg -q -Otest-image.jpg\n",
+        "!wget https://farm3.static.flickr.com/2142/1896267403_24939864ba.jpg -q -Otest-image2.jpg"
+      ],
+      "execution_count": 29,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Y-m7lYDPb4oK"
+      },
+      "source": [
+        "import matplotlib.pylab as plt\n",
+        "%matplotlib inline"
+      ],
+      "execution_count": 30,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "5bi8Q7L3b4oc",
+        "outputId": "e5de5c73-e16d-4992-ebee-94c7ddf0bf54",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 303
+        }
+      },
+      "source": [
+        "from torchvision.io.image import read_image\n",
+        "\n",
+        "tensor_image = read_image(\"test-image.jpg\")\n",
```

Comment:
I think it would be good to mention that torchvision now contains image reading functions for PNG and JPG images, with torchscript support, so that image decoding can be part of your model as well.